src/error/rust.rs

```diff
@@ -79,32 +86,43 @@ impl From<std::string::FromUtf8Error> for Error {
     }
 }
 
+/// cbindgen:prefix-with-name
+#[repr(i32)]
+enum ErrnoValue {
+    Success = 0,
+    Other = 1,
+    OutOfGas = 2,
+}
+
```

Comment:
The `OutOfGas` case here isn't really used on the Go side right now. I added it because I thought it would be useful, but I think that I ended up writing things in a way where if gas runs out, it will still trigger an "out of gas" in Go anyways, because it still tries to report gas usage, which will make the system notice we're overcharging. see [here](https://github.com/cosmos/cosmos-sdk/blob/4ffabb65a5c07dbb7010da397535d10927d298c1/store/types/gas.go#L79-L91)

Reply:
https://github.com/CosmWasm/wasmd/pull/115